### PR TITLE
[REFAC] rename bpf log, set Log helper

### PR DESCRIPTION
**Pull request by geyslan**
## Description (git log)

ace8a2ab logger: ebpf: convert BPFError to a Log helper

    Log is a helper that can be used, by instance, in cases where the
    log demand doesn't originate from user space.

1910af49 pkg/ebpf: rename bpf error to bpf log

Fixes: #2498 #2479

## Type of change

- [x] Code refactor (code improvement and/or code removal)

### CODE REVIEW

File: pkg/ebpf/bpf_log.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+func (b BPFLog) File() []byte {
+	return b.file[:]
+}
+
+func (b BPFLog) FileAsString() string {
+	errFile := string(b.file[:])
+	errFileEnd := strings.Index(errFile, "\x00")
+
+	return errFile[:errFileEnd]
```
Comment (yanivagman) [inline pkg/ebpf/bpf_log.go:122]: nit: shouldn't we change this to logFile?

File: pkg/ebpf/tracee.go [hunk 2]
```diff
@@ -170,42 +170,42 @@ type eventConfig struct {
...
+	netPerfMap         *bpf.PerfBuffer
+	bpfLogsPerfMap     *bpf.PerfBuffer
+	eventsChannel      chan []byte
+	fileWrChannel      chan []byte
+	netChannel         chan []byte
+	bpfLogsChannel     chan []byte
+	lostEvChannel      chan uint64
+	lostWrChannel      chan uint64
+	lostNetChannel     chan uint64
+	lostBPFLogsChannel chan uint64
```
Comment (yanivagman) [inline pkg/ebpf/tracee.go:188]: nit: You can change to lostBPFLogChannel to avoid the extra indentation

**Review by yanivagman — APPROVED** [review]
LGTM
nit comments ahead

**Review by NDStrahilevitz — APPROVED** [review]
LGTM

File: pkg/ebpf/bpf_log.go (hunk 1 above)
Comment (geyslan) [reply pkg/ebpf/bpf_log.go]: It's leftovers. Tks. Done.

**Pull request merged**